tests/spec/fs.utimes.spec.js

```diff
@@ -242,7 +242,7 @@ describe('fs.promises.utimes', function () {
     var atime = Date.parse('1 Oct 2000 15:33:22');
     var mtime = Date.parse('30 Sep 2000 06:43:54');
 
-    fs.utimes(1, atime, mtime)
+    return fs.utimes(1, atime, mtime)
```

Comment:
Not sure why all the tests passed when I forgot the return statements, but now that I added them just this one test is failing

Reply:
Should have called `futimes`, that's why it failed.

Reply:
@humphd, why do all the tests pass when I leave out the `return` statements?

Reply:
The test hits the end of the stack, and nothing failed, so it says, "OK, I guess this worked."  Meanwhile, your file system calls are running async, and haven't started (or finished) yet:

```js
it('...', function() {
  startSomethingAsync();
  //test will end right here, before startSomethingAsync can finish
});
```

However, when you `return` a promise (or with callbacks use `done`), you block the test from exiting until the task is complete (promise is resolved, or callback done() is executed).

Reply:
thank you!